test/src/core-layers/core-layers.spec.js

```diff
@@ -66,23 +66,6 @@ test('ScreenGridLayer#constructor', t => {
             layer.state.cellScale === layer.state.model.uniforms.cellScale,
             'should update uniform cellScale'
           );
-          t.ok(
```

Comment:
Why did we need to cut a test?

Reply:
As part of supporting `colorRange` and `colorDomain`, color calculation is moved out of the shader, so I am removing these un-used uniforms and their update, hence remove these unit tests.